### DISCUSSION TITLE
Using in SSL WebSocket

### DISCUSSION THREAD

**Discussion by EvandroMassini** [arthurkushman/php-wss#60] [Q&A]
I'm test in a normal WebSocket, its fine. But, I need a safe WebSocket, and this example not work. So, I download of source in gitHub, because in composer this is not compatible,  the "setIsSsl" function not exists in "ServerConfig.php". Then, I use the example of "Readme.md" (SSL Server run options), but I have a many problems of incomatiblities of types. Ex: "Parse error: syntax error, unexpected 'array' (T_ARRAY), expecting function (T_FUNCTION) or const (T_CONST) in...". This line is: "public array $pathParams = [];". So, you heave a idea of to this examples work fine? You make a great job. Thanks.

**Comment by arthurkushman** [comment]
Hi, please provide full stack trace of an error, provide branch/version from wich u've downloaded the source code

Also, what do u mean by: "in composer this is not compatible"?

**Reply by EvandroMassini** [reply]
Initially I use the composer to install this dependencies, but some funcions and propertyes not exits in the code. Ex: "setIsSsl" function not exists in "ServerConfig.php". So, I download full code source in git, then I having problem whit type of variables. But, today I verify the my PHP versionm is 7.3 (Xampp) and requeriment is 7.4. I will make a new test. Thanks for you response.
